homeassistant/components/netatmo/light.py

```diff
@@ -0,0 +1,146 @@
+"""Support for the Netatmo camera lights."""
+import logging
+
+import pyatmo
+
+from homeassistant.components.light import LightEntity
+from homeassistant.core import callback
+from homeassistant.exceptions import PlatformNotReady
+
+from .const import DATA_HANDLER, DOMAIN, MANUFACTURER
+from .data_handler import CAMERA_DATA_CLASS_NAME, NetatmoDataHandler
+from .netatmo_entity_base import NetatmoBase
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Netatmo camera light platform."""
+    if "access_camera" not in entry.data["token"]["scope"]:
+        _LOGGER.info(
+            "Cameras are currently not supported with this authentication method"
+        )
+        return
+
+    data_handler = hass.data[DOMAIN][entry.entry_id][DATA_HANDLER]
+
+    data_class_name = CAMERA_DATA_CLASS_NAME
```

Comment:
At least in this scope, we could replace all the `data_class_name` with `CAMERA_DATA_CLASS_NAME` since it never changes